data_steward/tools/add_hpo.py

```diff
@@ -363,6 +383,14 @@ def main(hpo_id, org_id, hpo_name, bucket_name, display_order, addition_type,
                         '--bucket_name',
                         required=True,
                         help='Name of the GCS bucket')
+    parser.add_argument('-s',
+                        '--us_state',
+                        required=True,
+                        help="Site's State as PIIState_XY.")
```

Comment:
You can directly call the check function here.
```suggestion
    parser.add_argument('-s',
                        '--us_state',
                        required=True,
                        type=check_state_code_format,
                        help="Site's State as PIIState_XY.")
```